packages/core/src/fund.ts

```diff
@@ -112,24 +115,20 @@ export const estimateExecutionGas = async (
 
   estimatedGas = estimatedGas.add(estimatedContractDeploymentGas)
 
-  return estimatedGas
+  return estimatedGas.add(BigNumber.from(150_000))
```

Comment:
Is there a reason you changed this from 400k? 150k seems sort of low since these functions have a for-loop and include the `_payExecutorAndProtocol` functions. Haven't tested it myself though

Reply:
Ya know, I actually decided to just get rid of this entirely [see here](https://github.com/chugsplash/chugsplash/pull/640/files#r1179801665).